src/main/fc/core.h

```diff
@@ -47,6 +47,19 @@ typedef enum {
     LAUNCH_CONTROL_MODE_COUNT // must be the last element
 } launchControlMode_e;
 
+typedef enum {
+    DISARM_REASON_SYSTEM            = 0,
```

Comment:
Code that is only used for tests needs to be gated with `UNIT_TEST`.

Reply:
Do you think it's acceptable to use `255` for the gated value so that the first 'real' item has the value of `0`?

Reply:
As stated above, I think it is preferable if tests test actual production code where possible, instead of testing test scaffolding - this maximises their efficiency in detecting regression. So ideally this should be testing with a value that is used in production code.
If you are insistent on having your tests test scaffolding code then moving it to 255 should be fine as long as it is marked as test scaffolding by being conditional to `UNIT_TEST`.

Reply:
ok, that'll work for me for now.  Agreed it would be good to have some assertions in the tests, but I'm currently pressed for time.